VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -259,12 +252,8 @@ public void onConfigurationChanged(Configuration newConfig) {
     private void updateUIForOrientation(){
         if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
             setActionBarVisible(false);
```

Comment:
@mdinino I don't think the toolbar should be hidden either (It's not hidden on iOS). So in fact, this whole method can be removed.